forte/data/readers/base_reader.py

```diff
@@ -222,16 +222,27 @@ def iter(self, *args, **kwargs) -> Iterator[PackType]:
         if self._cache_in_memory and self._cache_ready:
             # Read from memory
             for pack in self._data_packs:
+                self.record(pack._meta.record)
                 yield pack
         else:
             # Read via parsing dataset
             for pack in self._lazy_iter(*args, **kwargs):
+                self.record(pack._meta.record)
                 if self._cache_in_memory:
                     self._data_packs.append(pack)
                 yield pack
 
         self._cache_ready = True
 
+    def record(self, record_meta: Dict):
+        r"""Modify the datapack meta record field
+        Args:
+            record_meta: the field in the datapack for type record that need to fill in
+            for consistency checking.
```

Comment:
I think you need to have an indention if the docstring for an argument is longer than a line.

Reply:
Here is an example of wrong formatted docstring:
https://github.com/asyml/forte/blob/master/forte/data/data_pack.py#L290
https://asyml-forte.readthedocs.io/en/latest/code/data.html#forte.data.data_pack.DataPack.get_original_span

And here is an example of the correct format:
https://github.com/asyml/forte/blob/master/forte/data/data_pack.py#L414
https://asyml-forte.readthedocs.io/en/latest/code/data.html#forte.data.data_pack.DataPack.deserialize